judge/forms.py

```diff
@@ -137,3 +137,13 @@ def __init__(self, *args, **kwargs):
     def clean_totp_token(self):
         if not pyotp.TOTP(self.totp_key).verify(self.cleaned_data['totp_token'], valid_window=self.TOLERANCE):
             raise ValidationError(_('Invalid Two Factor Authentication token.'))
+
+
+class ProblemCloneForm(Form):
+    code = CharField(max_length=20)
+
+    def clean_code(self):
```

Comment:
You should also ensure that this problem code satisfies the regex defined [here](https://github.com/DMOJ/site/blob/master/judge/models/problem.py#L97).